src/main/scala/inox/ast/DSL.scala

```diff
@@ -65,7 +65,7 @@ trait DSL {
 
     def getField(selector: Identifier) = ADTSelector(e, selector)
 
-    def apply(es: Expr*) = Application(e, es.toSeq)
+    def apply(es: Expr*) = Application(e, es.toSeq).copiedFrom(e)
```

Comment:
I'm not sure it makes much sense to assign the position directly within the DSL, especially in such a one-of way. It would be better if we could assign the position where the DSL is used.

Reply:
I removed this, looks like we still have no missing positions in the benchmarks